src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -40,17 +40,31 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    /**
+    * Convert all strings in a set to lower case strings.
+    */
+    private Set<String> convertToLowerCase(Set<String> stringSet) {
```

Comment:
Do you think that moving this method into a util class will improve the SRP of the code?